postem/postem-app/.gitignore

```diff
@@ -0,0 +1 @@
+/.apt_generated_tests/
```

Comment:
Remove this file, we have a general gitignore.

Reply:
Done.